encode_test.go

```diff
@@ -12,7 +12,8 @@ import (
 	"os"
 
 	. "gopkg.in/check.v1"
-	"gopkg.in/yaml.v2"
+	//"gopkg.in/yaml.v2"
+	"github.com/go-yaml/yaml"
```

Comment:
not sure what to do about the imports, I would just leave them as they are for now.